staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/validation.go

```diff
@@ -79,6 +79,27 @@ func ValidateCustomResource(fldPath *field.Path, customResource interface{}, val
 				}
 				allErrs = append(allErrs, field.NotSupported(errPath, err.Value, values))
 
+			case openapierrors.TooLongFailCode:
+				value := interface{}("")
+				if err.Value != nil {
+					value = err.Value
+				}
+				allErrs = append(allErrs, field.TooLongFail(errPath, value, err.Error()))
+
+			case openapierrors.TooManyPropertiesCode, openapierrors.MaxItemsFailCode:
+				value := interface{}("")
+				if err.Value != nil {
+					value = err.Value
+				}
+				allErrs = append(allErrs, field.TooManyFail(errPath, value, err.Error()))
```

Comment:
kube-openapi is super inconsistent about what it places in `Value` for these errors... in some cases, it's the value, in some cases, it's the actual length, in some cases, it's the max length. The unit tests will reveal if propagating `value` into the field error makes for a sensible or weird error... if it's weird, omitting the value would be better than propagating a confusing value

Reply:
Created an issue in https://github.com/kubernetes/kube-openapi/issues/296 and will fix it later